libarchive/ffi.py

```diff
@@ -127,9 +127,12 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_int)
+ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_longlong)
```

Comment:
Sorry, I thought about this commit https://github.com/Changaco/python-libarchive-c/commit/f301a92e86a9660e0f6fa52139331de8a75b3d92
yes, it's completely unnecessary.